modules/apps/configuration-admin/configuration-admin-web/src/main/java/com/liferay/configuration/admin/web/internal/util/ConfigurationModelToDDMFormValuesConverter.java

```diff
@@ -201,6 +203,31 @@ else if (type.equals(DDMFormFieldType.SELECT)) {
 		ddmFormFieldValue.setValue(localizedValue);
 	}
 
+	private void _validateDDMFormValuesWithOverrideConfiguration(
+		DDMFormValues ddmFormValues) {
+
+		LocalizedValue localizedValue = new LocalizedValue();
+
+		Map<String, Object> configurationOverrideProperties =
+			_configurationModel.getConfigurationOverrideProperties();
+
+		for (DDMFormFieldValue ddmFormFieldValue :
+				ddmFormValues.getDDMFormFieldValues()) {
+
+			if (configurationOverrideProperties.containsKey(
+					ddmFormFieldValue.getName())) {
+
+				localizedValue.addString(
+					ddmFormValues.getDefaultLocale(),
+					configurationOverrideProperties.get(
+						ddmFormFieldValue.getName()
+					).toString());
```

Comment:
It is not good to chain methods (configurationOverrideProperties.get().toString()).
I thought about two options (but there is more):

1. GetterUtil.getString(configurationOverrideProperties.get(ddmFormFieldValue.getName()))
2. MapUtil.getString(configurationOverrideProperties, ddmFormFieldValue.getName())